BookingSystem/BookingDatabase/Script.SeedDatabase.sql

```diff
@@ -0,0 +1,85 @@
+﻿IF (NOT EXISTS(SELECT * FROM [dbo].[Users]))
+BEGIN
+    INSERT INTO [Users]
+			([FirstName], [SecondName], [Email], [PasswordHash], [PasswordSalt])
+	VALUES
+		('admin', 'admin', 'admin@admin.com', '1', '1'),
+		('user', 'user', 'user@user.com', '2', '1');
+END
+
+IF (NOT EXISTS(SELECT * FROM [dbo].[Hotels]))
+BEGIN
+	INSERT INTO [dbo].[Hotels]([Name],[Address],[CountryId], [CityId])
+	SELECT [Hotels].[Name], [Addresses].[Address], [Cities].[CountryId], [Cities].[CityId]
+	FROM (SELECT [Name], ROW_NUMBER() OVER (ORDER BY (SELECT 0)) AS row_num 
+		  FROM (VALUES ('Moderne Hotel'), ('The County'), ('Madrid Plaza')) AS [Hotels]([Name])) AS [Hotels]
+	JOIN 
+		(SELECT [CityId], [CountryId], ROW_NUMBER() OVER (ORDER BY (SELECT 0)) AS row_num 
+		 FROM [dbo].[Cities]) AS [Cities]
+	ON [Hotels].row_num = [Cities].row_num
+	JOIN 
+		(SELECT [Address], ROW_NUMBER() OVER (ORDER BY (SELECT 0)) AS row_num
+		 FROM 
+		 (VALUES ('243 West 55th St., New York, NY 10019'),
+		 		 ('83-95 Southampton Row, London WC1B 4HD'),
+				 ('Calle Gran Vía, 74 Madrid')) 
+		 AS [Addresses]([Address])) AS [Addresses]
+	ON [Addresses].row_num = [Hotels].row_num
+END
+
+
+DECLARE @FirstHotelId INT, @SecondHotelId INT;
+SET @FirstHotelId = (SELECT [HotelId] FROM [dbo].[Hotels] WHERE [Name] = 'Moderne Hotel');
```

Comment:
just FYI, the following statement is also possible:
SELECT@FirstHotelId = [HotelId] 
FROM [dbo].[Hotels] 
WHERE [Name] = 'Moderne Hotel'